components/Tabs/Tabs.js

```diff
@@ -41,18 +47,19 @@ class TabLink {
 class TabCard {
   constructor(element){
     // Assign this.element to the passed in element.
-    this.element;
+    this.element = tabs;
   }
   selectCard(){
     // Update the style of this.element to display = null
-    this.element;
+    this.element.display = null;
```

Comment:
You want to reach the _style_ of this element and then set its display to null. `this.element.style.display = null`.